scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -100,6 +100,10 @@ def net_config(args):
     return get_config(values, os.path.join(TEMPLATE_DIR, "admin-net.xml"))
 
 
+def merge_dicts(d1, d2):
+    return dict(it.chain(d1.items(), d2.items()))
```

Comment:
I meant to say `return d1`. These variables are having a very short lived scope, copy() step is quite unnecessary and mutating d1 is not a sacrifice IMHO!